File: Praktikum 1/Bahasa Python/PRAK104-2210817210028-Muhammad Aulia Rasyid.py
from hashlib import md5


A = 400000
B = 350000
diskonA = A * 13 / 100
hargaA = A - diskonA
hargaA1 = round(hargaA)
diskonB = B * 21 / 100
hargaB = B - diskonB
hargaB1 = round(hargaB)
print("Harga sepatu A adalah", A)
print("Harga sepatu B adalah", B)
print("Sepatu A mendapat diskon 13% sehingga harganya menjadi", hargaA1)
print("Sepatu B mendapat diskon 21% sehingga harganya menjadi", hargaB1)

File: Praktikum 1/Bahasa C/PRAK104-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>

int main()
{
    float A , B , diskonA , diskonB , hargaA , hargaB;
    char modulus = '%';
    A = 400000;
    B = 350000;
    diskonA = A * 13 / 100;
    hargaA = A - diskonA;
    diskonB = B * 21 / 100;
    hargaB = B - diskonB;
    printf("Harga sepatu A adalah %.0f\n", A);
    printf("Harga sepatu B adalah %.0f\n", B);
    printf("Sepatu A mendapat diskon 13%c sehingga harganya menjadi %.0f\n", modulus , hargaA);
    printf("Sepatu B mendapat diskon 21%c sehingga harganya menjadi %.0f\n", modulus , hargaB);
    return 0;
}
